rpm/authselect.spec.in

```diff
@@ -321,6 +321,16 @@ if [ -f %{forcefile} ]; then
     %__rm -f %{forcefile}
 fi
 
+# Minimal profile was removed. Switch to local.
+%__sed -i '1 s/^minimal$/local/'  %{_sysconfdir}/authselect/authselect.conf
```

Comment:
Add a suffix to create a backup?

Reply:
Nah, I don't think it it needed here. It would just create a file that would be on the system forever. If by any chance it gets corrupted, users will run "authselect select --force" anyway instead of restoring from a backup file that hey are not aware of (and that has outdated content so they will have to run select anyway).